dkg/dkg.go

```diff
@@ -635,3 +645,19 @@ func dvsFromShares(shares []share) []cluster.DistValidator {
 
 	return dvs
 }
+
+// writeLockToAPI posts the lock file to obol-api.
+func writeLockToAPI(ctx context.Context, publish string, lock cluster.Lock) error {
+	cl := obolapi.New(publish)
+	if err := cl.VerifyConnection(ctx); err != nil {
+		return err
+	}
+
+	if err := cl.PublishLock(ctx, lock); err != nil {
+		return err
+	}
+
+	log.Debug(ctx, "Published lock file to api")
```

Comment:
this can be info level, suggest `Published lock file, z.Str("addr", publish_addr)`